funnel/views/proposal.py

```diff
@@ -398,9 +406,15 @@ def proposal_moveto(profile, space, proposal):
         abort(404)
     profile_name, space_name = target_name.split('/', 1)
     target_space = ProposalSpace.query.filter(ProposalSpace.name == space_name).join(Profile).filter(Profile.name == profile_name).one_or_none()
-    if target_space:
-        proposal.move_to(target_space)
-        db.session.commit()  # WARNING: GET request!
-        return redirect(proposal.url_for(), 303)
-    else:
+
+    if not target_space:
         abort(404)
+
+    if not (target_space.profile.admin_team in g.user.teams or target_space.admin_team in g.user.teams):
+        flash(_("You do not have permission to move this proposal to {space}.".format(space=target_space.title)))
+        abort(403)
+
+    proposal.move_to(target_space)
+    db.session.commit()  # WARNING: GET request!
```

Comment:
Is this still a GET request?

Reply:
Yes. I was just think about making it a PUT or POST request.